camel/societies/role_playing.py

```diff
@@ -242,8 +242,7 @@ def get_sys_message_info(
         sys_msg_meta_dicts = [dict(task=self.task_prompt) for _ in range(2)]
         if (extend_sys_msg_meta_dicts is None and self.task_type
                 in [TaskType.AI_SOCIETY, TaskType.MISALIGNMENT]):
-            if (assistant_description is not None
-                    and user_description is not None):
+            if (assistant_description != "" and user_description != ""):
```

Comment:
Do not use empty strings. It was good before, with Optional. Change to None and defaults to None as well.